### ISSUE TITLE
Battery capacity script telemetry reset.

### ISSUE THREAD

**Issue by SkyPod** [RCdiy/OpenTxLua#5]
Hi all
Has anyone worked with RCdiy battery capacity script? Its an awesome little program and it can be very accurate. Where I'am looking for help is I can't get the script not to reset the mAh used after a telemetry reset. I need to reset the telemetry to get the altimeter set to 0 feet and this may be after working on something for a few minutes.  I'am really frustrated. Having tried many changes but not knowing Lua script very well has not helped.  Have spent many hours on this any help is greatly appreciated.
Thanks SkyPod

**Comment by RCdiy** [comment]
Are you still having a problem?

**Comment by SkyPod** [comment]
Yes. I just gave up on it. It would be very accurate to have it continuously run even after a reset.  Really thankful for all your videos they've been my go to for a while.

**Comment by RCdiy** [comment]
For milliampere hours the consumption sensor is used. If that resets then then the values go back to the start. How much do you use before doing a reset. Is it even 1%

**Comment by SkyPod** [comment]
Thanks For the reply.  I am away from all my RC stuff till next Friday. I
can look at it then and let you know.
I really appreciate you reaching out.

**Comment by SkyPod** [comment]
Hello
Jest now having time to work with this. Yes setting up my ground station and other things dose take a few % of the battery life. And it dose reset back to 0%. It would be nice to have it as accurate as possible. Any help on this would be greatly appreciated.
Thanks Raymond
